src/xcape/core.py

```diff
@@ -290,7 +349,16 @@ def _calc_srh_gufunc(*args, **kwargs):
 # the numpy version of the algorithm
 def _calc_srh_numpy(*args,
                     depth = 3000, vertical_lev='sigma',output_var='srh'):    
-    
+    ''' 
+     Wrapper function for SRH calculation to setup optional kwargs and 
+     ensure data is provided in a format suitable output to call the 
+     Fortran implementation of SRH calculation. Also calculates standard
+     height using the hypsometric equation, which necessitates the passing
+     of pressure, temperature and dewpoint temperature. Called by 
+     _calc_srh_gufunc.   
+     '''    
+
```

Comment:
maybe add "calculates standard height" right before the call of:
aglh_2d, aglh_s1d = _stdheight(p_2d, t_2d, td_2d,

below